lua/telescope/make_entry.lua

```diff
@@ -452,37 +452,25 @@ function make_entry.gen_from_quickfix(opts)
   local show_line = vim.F.if_nil(opts.show_line, true)
 
   local hidden = utils.is_path_hidden(opts)
-  local items = {
-    { width = vim.F.if_nil(opts.fname_width, 30) },
```

Comment:
This means no more `fname_width` option for anything that uses this entry maker right?

Probably should remove that option from the docs (docstring in `builtin/init.lua`).